CMakeLists.txt

```diff
@@ -810,6 +810,38 @@ if (ENABLE_KERNELGETMEMORYSIZE)
   message("-- Memory: Kernel_GetMemorySize() Enabled.")
 endif()
 
+# Set MPI args
```

Comment:
Since you mived them here, we can prob delete these settings down below (near file end).

Reply:
Whoops.  Yeah, I meant to do that.  Good catch.

Reply:
@bartgol Fixed with faae04967b87bcea753e79881861463321f0e247.